tests/test_interestingness.py

```diff
@@ -21,7 +21,7 @@
 
 # The following test cases are labelled for vis with <Ndim, Nmsr, Nfilter>
 def test_interestingness_1_0_0(global_var):
-    df = pytest.car_df
+    df = pd.read_csv("lux/data/car.csv")
```

Comment:
Is there a reason why all of these were switched back to `read_csv`?

Reply:
Yah, since it's the same DataFrame being used across different tests, the History is propagated through, so if one test did a groupby, all future tests will have groupby in the history of the `df`. So, for the ones with a groupby, we had to load in the data directly.